app/assets/stylesheets/base/_fonts.scss

```diff
@@ -1,5 +1,5 @@
 // This file contains the base fonts for the application.
 
-@import url(https://fonts.googleapis.com/css?family=Lato:100,400,300);
+@import url(https://fonts.googleapis.com/css?family=Roboto:100,400,300);
 
-$font-family: 'Lato';
+$font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', sans-serif;
```

Comment:
We want to use Lato as the default though. We could fallback to another font if it's in Vietnamese though.